test/index.js

```diff
@@ -0,0 +1,15 @@
+import assert      from 'assert'
+import superagent  from 'superagent'
+import startServer from '../lib/server'
+
+startServer(8001)
```

Comment:
What would the URL for testing be, then? Would I just not need one, due to server.inject?